airbyte-webapp/src/components/base/Input/Input.tsx

```diff
@@ -109,8 +106,14 @@ const Input: React.FC<InputProps> = (props) => {
         ref={inputRef}
         type={type}
         isPassword={isPassword}
-        onFocus={onInputFocusChange}
-        onBlur={onInputFocusChange}
+        onFocus={(event) => {
+          toggleFocused();
```

Comment:
Since we're touching this right now: Could we maybe change this `useToggle` into a `useState` and actually use `onFocus`/`onBlur` to set the focused state correctly. Most of the times it shouldn't be a problem, but I've encountered in the past some weird edge-cases where `onFocus`/`onBlur` might be called twice (e.g. when browser window focus changes in a weird way), in which case the toggle solution might lose correct track of actual focus.